app/views/annotations/_annotation_list.html.erb

```diff
@@ -1,9 +1,17 @@
 <ul class='annotation_text_list'>
-  <% if annotation_category.annotation_texts.empty? %>
+  <%
+    text = []
+    annotation_category.annotation_texts.each do |annotation_text|
+      if (annotation_text.user.nil? || (annotation_text.user.admin? || annotation_text.user == @current_user))
```

Comment:
Ah, good catch. This is an error in the development environment seeding (please fix); we never generate global annotations like that in production. In fact, it would be great if you added a validation for `user` for both category and annotation.